scripts/maintenance/migrate_project/src/r_clone.py

```diff
@@ -0,0 +1,78 @@
+from pathlib import Path
+from subprocess import PIPE, Popen  # nosec
+
+DESTINATION = "dst"
+SOURCE = "src"
+
+CONFIG = """
+[{destination}]
+type = s3
+provider = AWS
+access_key_id = {aws_access_key}
+secret_access_key = {aws_secret_key}
+region = us-east-1
+acl = private
+
+[{source}]
+type = s3
+provider = Minio
+access_key_id = {minio_access_key}
+secret_access_key = {minio_secret_key}
+endpoint = {minio_endpoint}
+region = us-east-1
+acl = private
+"""
+
+
+def assemble_config_file(
+    aws_access_key: str,
+    aws_secret_key: str,
+    minio_access_key: str,
+    minio_secret_key: str,
+    minio_endpoint: str,
+) -> Path:
+    # NOTE: Since rclone requires slightly different configuration based on the
+    # S3 provider, below assumptions are made:
+    # - source: MINIO
+    # - destination: AWS S3
+    # The above CONFIG will require changing if this changes
+    config_content = CONFIG.format(
```

Comment:
why not using ``string.Template`` ... i learned actually thatone with one of your PRs. I think it is the closest thing to jinja2 within the stdlib?

If you combine with the Settings, this whole files reduced to a few lines